dftimewolf/lib/processors/stackdriver_timesketch.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+"""Processes Stackdriver logs for loading into Timesketch."""
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import tempfile
+import json
+
+from dftimewolf.lib.module import BaseModule
+from dftimewolf.lib.containers import StackdriverLogs
+
+
+class StackdriverTimesketch(BaseModule):
+  """Transforms Stackdriver logs for Timesketch."""
+
+  def __init__(self, state):
+    super(StackdriverTimesketch, self).__init__(state)
+
+  def setup(self, *args, **kwargs):
+    """Sets up necessary module configuration options."""
+
+  def cleanup(self):
+    """Cleans up module output to prepare it for the next module."""
+
+  def _process_log_line(self, log_line, query, project_name):
+    """Processes a single JSON formatted Stackdriver log.
+
+    Args:
+      log_line (str): a JSON formatted stackdriver log entry.
+      query (str): the Stackdriver query used to retrieve the log.
+      project_name (str): name of the GCP project associated with the query.
+
+    Returns:
+      str: a Timesketch-friendly version of the log line.
+    """
+    log_record = json.loads(log_line)
+
+    # Metadata about how the record was obtained.
+    timesketch_record = {'query': query, 'project_name': project_name}
+
+    # Timestamp related fields.
+    timestamp = log_record.get('timestamp', None)
+    if timestamp:
+      timesketch_record['datetime'] = timestamp
+      timesketch_record['timestamp_desc'] = 'Event Recorded'
+
+    # General resource information.
+    resource = log_record.get('resource', None)
+    if resource:
+      labels = resource.get('labels', None)
+      if labels:
+        for attribute, value in iter(labels.items()):
```

Comment:
is `iter` necessary here?

Reply:
Only for Python 2 compatibility, which doesn't matter here.